README.md

```diff
@@ -1,4 +1,32 @@
-# sarif-pattern-matcher
+<p align="center"><img src="Src/img/header_img.png" width="440" /></p>
 
-Quality domain agnostic regular expression pattern matcher that persists results
-to SARIF
+<p align="center">
+
+  <a href="https://github.com/microsoft/sarif-pattern-matcher">
+    <img src="https://img.shields.io/github/v/release/microsoft/sarif-pattern-matcher"
+      alt="release" />
+  </a>
+  
+  <a href="https://github.com/microsoft/sarif-pattern-matcher">
+    <img src="https://img.shields.io/github/v/release/microsoft/sarif-pattern-matcher?include_prereleases&label=pre-release"
+      alt="pre-release" />
+  </a>
+  	
+ <a href="https://github.com/microsoft/sarif-pattern-matcher/blob/master/LICENSE">
+    <img src="https://img.shields.io/github/license/microsoft/sarif-pattern-matcher"
+      alt="License: MIT" />
+  </a>
+  
+  
+</p>
+
+<p align="center">Quality domain agnostic regular expression pattern matcher that persists results to SARIF</p>
+
+## How To Contribute
+
+TypeScript-Handbook is accepting contributions. If you've submitted a PR for an existing issue, please post a comment in the issue to avoid duplication of effort. See our [CONTRIBUTING](/CONTRIBUTING.md) file for more information - it also contains guidelines for how to submit a PR.
```

Comment:
Should change to sarif pattern matcher.

Reply:
Ya that my mistake i just copy from my last project i update that